universal-application-tool-0.0.1/app/views/admin/versions/VersionListView.java

```diff
@@ -53,7 +53,8 @@ public Content render(List<Version> allVersions, Http.Request request) {
     ImmutableList<Version> olderVersions =
         allVersions.stream()
             .filter(version -> version.getLifecycleStage().equals(LifecycleStage.OBSOLETE))
-            .collect(ImmutableList.toImmutableList());
+            .collect(ImmutableList.toImmutableList())
+            .reverse();
```

Comment:
suggestion: We likely need to explicitly sort here too.  The method does not require they be passed in sorted or in any particular order, and it's likely a matter of happenstance that they are ordered and in a particular way that we want to arbitrarily reverse.  So I think we need to sort here too.

Reply:
added explicit sorting